shap/explainers/_tree.py

```diff
@@ -155,7 +158,17 @@ def __init__(
             Currently the "probability" and "log_loss" options are only
             supported when ``feature_perturbation="interventional"``.
 
+        approximate : bool
+            Deprecated, will be removed in version v0.48.0. Please use the ``approximate`` argument in the
```

Comment:
May I suggest a slightly longer deprecation period - how about giving 2 versions' notice (`v0.49.0`), which I think is somewhat standard?

Also, may I suggest adding two entries to our tracker #3507:
- Deprecating the param in 0.47
- Removing the param in 0.49 (?)

Reply:
done